src/test/java/strategy/AddOperationHandlerTest.java

```diff
@@ -0,0 +1,45 @@
+package strategy;
+
+import db.Storage;
+import java.util.Map;
+import model.Fruit;
+import model.TransactionDto;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class AddOperationHandlerTest {
+    private static TransactionDto transactionDto = new TransactionDto("b","banana", 100);
+    private static OperationHandler operator = new AddOperationHandler();
+    private static Fruit fruit = new Fruit(transactionDto.getFruitName());
+    private static Map<Fruit, Integer> actual = Storage.storage;
+    private static Map<Fruit, Integer> expected = actual;
```

Comment:
```suggestion
    private static OperationHandler operator = new AddOperationHandler();
```

все остальное нужно сделать локальными переменными (объявлять их в методе)
сейчас по-прежнему сложно очень читать код